lib/practice_exercises.rb

```diff
@@ -1,13 +1,36 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n), linear,  because the amount of times the while loop iterates is directly proportionate to the length of the array.
```

Comment:
Since you are using `delete_at` which has a runtime of O(n), you actually have O(n<sup>2</sup>) runtime complexity.

Remember `delete_at` shifts each of the subsequent elements over one index.